src/lumigo_opentelemetry/resources/detectors.py

```diff
@@ -102,13 +104,20 @@ def detect(self) -> "Resource":
 
 def get_resource(attributes: dict) -> "Resource":
     return get_aggregated_resources(
-        detectors=[
-            OTELResourceDetector(),
-            EnvVarsDetector(),
-            ProcessResourceDetector(),
-            LumigoDistroDetector(),
-            AwsEcsResourceDetector(),
-            LumigoAwsEcsResourceDetector(),
-        ],
+        detectors=_get_detector_list(),
         initial_resource=Resource.create(attributes=attributes),
     )
+
+
+def _get_detector_list() -> List[ResourceDetector]:
+    logging.getLogger("opentelemetry.sdk.extension.aws.resource.ecs").setLevel(
```

Comment:
check if `getLogger` returns optional - so that `.setLevel.` won't raise

Reply:
if the logger does not exist the getLogger will create it